site/app/models/gradeable/GradedGradeable.php

```diff
@@ -49,15 +49,30 @@ class GradedGradeable extends AbstractModel {
      * @var bool|null|SimpleGradeOverriddenUser Does this graded gradeable have overridden grades */
     protected $overridden_grades = false;
 
+    /** @prop
+     * @var array<array<int, string>> The active graders for this graded gradeable */
+    protected $active_graders_names = [];
```

Comment:
Are default values here necessary with the constructor parameters being required?